http_service/bugbug_http/app.py

```diff
@@ -190,7 +190,29 @@ def schedule_job(
     )
 
 
-def schedule_bug_classification(model_name: str, bug_ids: Sequence[int]) -> None:
+def prepare_queue_job(job: JobInfo, job_id: Optional[str] = None) -> Queue:
+    """Prepare queue data for for enqueueing multiple jobs"""
+    job_id = job_id or get_job_id()
+
+    redis_conn.mset({job.mapping_key: job_id})
+    return Queue.prepare_data(
+        job.func,
+        args=job.args,
+        job_id=job_id,
+        timeout=BUGZILLA_JOB_TIMEOUT,
+        ttl=QUEUE_TIMEOUT,
+        failure_ttl=FAILURE_TTL,
+    )
+
+
+def schedule_multiple_job(jobList: list[Queue] = None) -> None:
+    """Enqueue multiple jobs in bulk"""
+    q.enqueue_many(jobList)
```

Comment:
This function is not necessary, since it's simply calling another function without doing anything else

Reply:
removed the function